README.me

```diff
@@ -0,0 +1,23 @@
+1. Required python modules:
+
+```
+sudo apt install python-pip
+git clone https://github.com/ganehag/pyMeterBus
+cd pyMeterBus && sudo python setup.py install && cd ..
+sudo pip install paho-mqtt
```

Comment:
Note to self, probably better having paho-mqtt requirements.txt file and then sudo pip install -r requirements to follow standard practice.